examples/StandardFirmata/StandardFirmata.ino

```diff
@@ -20,13 +20,20 @@
 
   See file LICENSE.txt for further informations on licensing terms.
 
-  Last updated by Jeff Hoefs: August 9th, 2015
+  Last updated by Jeff Hoefs: October 3rd, 2015
 */
 
 #include <Servo.h>
 #include <Wire.h>
+#include <ArduinoUnit.h>
 #include <Firmata.h>
 
+// SoftwareSerial is only supported for AVR-based boards
+#if defined(ARDUINO_ARCH_AVR)
```

Comment:
The think we are trying to avoid here is including SoftwareSerial when complining a non-AVR board. This is because the SoftwareSerial is only available for AVR boards so if a user attempts to compile for a SAM board (Arduino Due, Teensy, etc) or other non-AVR architecture, then compilation will fail because there will be no SoftwareSerial.h library to be found.